[2023-12-27T23:39:49.412+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: KOH.data_ml manual__2023-12-27T18:39:39.397051+00:00 [queued]>
[2023-12-27T23:39:49.445+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: KOH.data_ml manual__2023-12-27T18:39:39.397051+00:00 [queued]>
[2023-12-27T23:39:49.445+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-27T23:39:49.549+0500] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): data_ml> on 2023-12-27 18:39:39.397051+00:00
[2023-12-27T23:39:49.555+0500] {standard_task_runner.py:57} INFO - Started process 6316 to run task
[2023-12-27T23:39:49.571+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'KOH', 'data_ml', 'manual__2023-12-27T18:39:39.397051+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/text.py', '--cfg-path', '/tmp/tmp3g509_pd']
[2023-12-27T23:39:49.576+0500] {standard_task_runner.py:85} INFO - Job 191: Subtask data_ml
[2023-12-27T23:39:49.702+0500] {task_command.py:416} INFO - Running <TaskInstance: KOH.data_ml manual__2023-12-27T18:39:39.397051+00:00 [running]> on host vm2.myguest.virtualbox.org
[2023-12-27T23:39:49.937+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Admin' AIRFLOW_CTX_DAG_ID='KOH' AIRFLOW_CTX_TASK_ID='data_ml' AIRFLOW_CTX_EXECUTION_DATE='2023-12-27T18:39:39.397051+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-27T18:39:39.397051+00:00'
[2023-12-27T23:39:49.938+0500] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-27T23:39:49.944+0500] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/kda/ML_lab3/MLops_lab3/ && python /home/kda/ML_lab3/MLops_lab3/scripts/model_learn.py']
[2023-12-27T23:39:49.954+0500] {subprocess.py:86} INFO - Output:
[2023-12-27T23:40:07.909+0500] {subprocess.py:93} INFO - Registered model 'lr' already exists. Creating a new version of this model...
[2023-12-27T23:40:08.113+0500] {subprocess.py:93} INFO - 2023/12/27 23:40:08 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: lr, version 22
[2023-12-27T23:40:08.114+0500] {subprocess.py:93} INFO - Created version '22' of model 'lr'.
[2023-12-27T23:40:08.304+0500] {subprocess.py:93} INFO - <class 'pandas.core.frame.DataFrame'>
[2023-12-27T23:40:08.304+0500] {subprocess.py:93} INFO - RangeIndex: 704 entries, 0 to 703
[2023-12-27T23:40:08.304+0500] {subprocess.py:93} INFO - Columns: 220 entries, Price (in USD) to i8
[2023-12-27T23:40:08.305+0500] {subprocess.py:93} INFO - dtypes: bool(212), float64(2), int64(6)
[2023-12-27T23:40:08.305+0500] {subprocess.py:93} INFO - memory usage: 189.9 KB
[2023-12-27T23:40:09.409+0500] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-27T23:40:09.536+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=KOH, task_id=data_ml, execution_date=20231227T183939, start_date=20231227T183949, end_date=20231227T184009
[2023-12-27T23:40:09.621+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-27T23:40:09.664+0500] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
